stable/yugaware/values.yaml

```
@@ -114,8 +114,11 @@ tls:
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
   enabled: false
-  fsGroup: 1001
-  runAsUser: 1001
+  fsGroup: 10001
+  runAsUser: 10001
+  runAsGroup: 10001
+  runAsNonRoot: true
+  fsGroupChangePolicy: "OnRootMismatch"
```

```suggestion
  ## fsGroup related values are set at the pod level.
  fsGroup: 10001
  fsGroupChangePolicy: "OnRootMismatch"
  ## The following values are set for yugaware and prometheus
  ## containers.
  runAsUser: 10001
  runAsGroup: 10001
  runAsNonRoot: true
```
This will change later, but for now it represents current state.